docker-compose.yml

```diff
@@ -61,8 +61,8 @@ services:
     command: bash -c 'sleep 20 && mongo -u admin -p admin --authenticationDatabase admin dbzui-db-mongo:27017/inventory --eval "rs.initiate();"'
   dbzui-connect:
     container_name: dbzui-connect
-    build: 
-      context: connect
+    image: 
+      debezium/connect:nightly
```

Comment:
@mdrillin can you maybe also please add the env var to enable scripting? some SMTs are in that package and it is needed for some functionality of Debezium SMTs.
compare: https://github.com/debezium/debezium-ui/blob/main/backend/src/test/java/io/debezium/configserver/util/Infrastructure.java#L66

Reply:
@rk3rn3r - I amended the commit to add the scripting env var.  Here is a screenshot of the transformation types I'm getting now:
![TransformTypes](https://user-images.githubusercontent.com/1820403/142442089-b116396b-b314-4a7a-9e54-d6bf43cae3a1.png)

Reply:
LGTM!